src/katsdpcontroller/product_controller.py

```diff
@@ -1628,14 +1628,18 @@ async def _launch_telstate(self) -> katsdptelstate.aio.TelescopeState:
                 for stream in itertools.chain(
                     self.configuration.by_class(product_config.AntennaChannelisedVoltageStream),
                     self.configuration.by_class(product_config.SimAntennaChannelisedVoltageStream),
+                    self.configuration.by_class(
+                        product_config.GpucbfAntennaChannelisedVoltageStream
+                    ),
                 ):
                     ratio = round(stream.adc_sample_rate / 2 / stream.bandwidth)
                     band_mask_model_urls = await _resolve_model(
                         fetcher,
                         model_base_url,
                         f"band_mask/current/{stream.band}/nb_ratio={ratio}.alias",
                     )
-                    prefix: Tuple[str, ...] = (stream.name, "model", "band_mask")
+                    stream_name_norm = stream.name.replace("-", "_")
```

Comment:
I see a few of these popped up recently. Is it too early to consolidate them in one place (e.g. `stream.normalised_name`)?